.github/workflows/code-quality.yml

```diff
@@ -35,3 +35,5 @@ jobs:
       run: flake8 pippy
     - name: Lint with black for SPMD
       run: black --check --line-length 80 spmd test/spmd
+    - name: Pyre check for SPMD
```

Comment:
nit: run pyre check only in spmd folder

Reply:
Pyre uses the `.pyre_configuration` file to identify target folders. And I have set it to include `spmd`, `test/spmd` only.